src/test/java/com/conveyal/datatools/manager/controllers/api/FeedSourceControllerTest.java

```diff
@@ -17,42 +18,31 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 
+import java.io.IOException;
+import java.net.URL;
+
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class FeedSourceControllerTest {
+public class FeedSourceControllerTest extends DatatoolsTest {
     private static Project project = null;
     private static FeedSource feedSourceWithUrl = null;
     private static FeedSource feedSourceWithNoUrl = null;
 
     @BeforeClass
-    public static void setUp() throws Exception {
-        // start server if it isn't already running
+    public static void setUp() throws IOException {
         DatatoolsTest.setUp();
+        Auth0Connection.setAuthDisabled(true);
         project = new Project();
         project.name = "ProjectOne";
         project.autoFetchFeeds = true;
         Persistence.projects.create(project);
-
-        feedSourceWithUrl = new FeedSource();
-        feedSourceWithUrl.fetchFrequency = FetchFrequency.MINUTES;
-        feedSourceWithUrl.fetchInterval = 1;
-        feedSourceWithUrl.deployable = false;
-        feedSourceWithUrl.name = "FeedSourceOne";
-        feedSourceWithUrl.projectId = project.id;
-        feedSourceWithUrl.retrievalMethod = FeedRetrievalMethod.FETCHED_AUTOMATICALLY;
-        feedSourceWithUrl.url = new java.net.URL("http://www.feedsource.com");
-
-        feedSourceWithNoUrl = new FeedSource();
-        feedSourceWithNoUrl.fetchFrequency = FetchFrequency.MINUTES;
-        feedSourceWithNoUrl.fetchInterval = 1;
-        feedSourceWithNoUrl.deployable = false;
-        feedSourceWithNoUrl.name = "FeedSourceTwo";
-        feedSourceWithNoUrl.projectId = project.id;
-        feedSourceWithNoUrl.retrievalMethod = FeedRetrievalMethod.FETCHED_AUTOMATICALLY;
+        feedSourceWithUrl = createFeedSource("FeedSourceOne", new URL("http://www.feedsource.com"));
+        feedSourceWithNoUrl = createFeedSource("FeedSourceTwo", null);
     }
 
     @AfterClass
     public static void tearDown() {
+        Auth0Connection.getDefaultAuthDisabled();
```

Comment:
This should be `Auth0Connection.setAuthDisabled(Auth0Connection.getDefaultAuthDisabled());`

Reply:
Fixed.